modules/ROOT/pages/enterprise/server/troubleshoot.adoc

```diff
@@ -132,19 +137,19 @@ Once downloaded:
 
```

Comment:
The list below is not rendering correctly (each item is a new list) because the images need to be linked to the list item. e.g:
```
. Unzip the package like so:
+
image::spell-checking-curl-windows-1.png[Curl on Windows step 1]
```

Reply:
done